gtsfm/view_graph_estimator/view_graph_estimator_base.py

```diff
@@ -58,6 +58,28 @@ def run(
             Edges of the view-graph, which are the subset of the image pairs in the input args.
         """
 
+    def _get_valid_input_edges(self, i2Ri1_dict: Dict[Tuple[int, int], Rot3]) -> List[Tuple[int, int]]:
```

Comment:
temp fix until Akshay's fix lands

Reply:
Should `i2Ri1_dict` be `Dict[Tuple[int, int], Optional[Rot3]]` since you check for `i2Ri1 is None` in the function?